### ISSUE TITLE
Bug: `no-unnecessary-whitespace` reports empty strings

### ISSUE THREAD

**Issue by andreww2012** [schoero/eslint-plugin-readable-tailwind#188]
I know this is not well written code, at least for Vue, yet I think empty strings should be allowed:

```html
<!-- error  Unnecessary whitespace  better-tailwindcss/no-unnecessary-whitespace -->
<div :class="[someCondition ? 'bg-red-100' : '']">Test</div>
```

Reproduction: https://stackblitz.com/edit/vitejs-vite-hsafjqbj

**Comment by jh-aimpera** [comment]
+1
